docs/source/_static/files/requirements_DeepRC.txt

```diff
@@ -0,0 +1,2 @@
+git+https://github.com/ml-jku/DeepRC@fec4b4f4b2cd70e00e8de83da169560dec73a419
```

Comment:
we can just link to this file from github - no need to duplicate? like this: https://raw.githubusercontent.com/uio-bmi/immuneML/master/requirements_DeepRC.txt

Reply:
great idea!